pages/dkp/kommander/2.2/release-notes/2.2.1/index.md

```diff
@@ -0,0 +1,123 @@
+| Prometheus Adapter      | prometheus-adapter                             | 2.17.1  | - chart: 2.17.1 `<br>`- prometheus-adapter: 0.9.1                                                                                                   |
+| Reloader                | reloader                                       | 0.0.104 | - chart: 0.0.104 `<br>`- reloader: 0.0.104                                                                                                          |
+| Thanos                  | thanos                                         | 0.4.6   | - chart: 0.4.6 `<br>`- thanos: 0.9.0                                                                                                                |
+| Traefik                 | traefik                                        | 10.9.1  | - chart: 10.9.1 `<br>`- traefik: 2.5.6                                                                                                              |
+| Traefik ForwardAuth     | traefik-forward-auth                           | 0.3.6   | - chart: 0.3.6 `<br>`- traefik-forward-auth: 3.1.0                                                                                                  |
+| Velero                  | velero                                         | 3.2.0   | - chart: 3.2.0 `<br>`- velero: 1.5.2                                                                                                                |
+
+## Known Issues
+
+The following items are known issues with this release.
+
+### Unsupported installation parameters
+
+If you set any these parameters in `kommander-vars` ConfigMap, the upgrade process fails.
+
+-   appManagementImageTag
+-   appManagementImageRepository
+-   kommanderChartsVersion
+
+Setting kommander chart version during installation configuration results in setting above values in kommander-vars config map, which then can block the upgrade. To unblock the upgrade, you can edit and remove the Config map values.
+Starting with release 2.2.0, you can specify the kommander chart version during installation configuration. However, if you manually change the value, then you might not be able to upgrade to higher versions.
+
+We recommend you DO NOT change the default value of the kommander chart version during installation. If you change this value, the outcome of such an installation is unpredictable. kommander-vars can be edited but it will cause an immediate restart to kommander 2.2.0. Without removing these vars, kommander cannot be upgraded.
+<QUESTION: do we want to recommend to contact support if this situation occurs and the user is blocked from upgrading?>
+
+## Additional resources
+
+<!-- Add links to external documentation as needed -->
+
+For more information about working with native Kubernetes, see the [Kubernetes documentation][kubernetes-doc].
+
```

Comment:
```suggestion
For a full list of attributed 3rd party software, see [d2iq.com/legal/3rd](http://d2iq.com/legal/3rd).
```
Since this has the purple download Kommander button on it, do we need the legal attribution for MinIO here at the bottom?